solutions/usaco-988.mdx

```diff
@@ -2,11 +2,14 @@
 id: usaco-988
 source: USACO Bronze 2020 January
 title: Photoshoot
-author: Ananth Kashyap
+author: Ananth Kashyap, Ben Dodge
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
 
+We'll start by saying that for each $i$ in range $[1, N]$, $a_1 = i$. For all other values of $a_i$, we are given $b_i = a_i + a_{i+1}$ for $[1, N)$, or $a_i = b_{i -1} - a_{i - 1}$ for $(1, N]$. We will then check if these values of $a_i$ produce a valid permutation. 
+
+
 Since $N \leq 1000$, we can simulate every ordering and pick the first valid permutation (since that would be the lexicographical minimum).
```

Comment:
you didn't add this part, but it's not really simulating every ordering? maybe change it to "simulate picking every cow as the first one" (but worded better)?